apps/vanilla/src/pages/index.html

```diff
@@ -6,11 +6,136 @@
     <link rel="stylesheet" href="./index.css" />
     <title>Vite App</title>
   </head>
-  <body class="default-layout">
-    <h1 class="title">Main page ⚡</h1>
-    <div class="links">
-      <a href="example/">Go to example page</a>
-    </div>
-    <script type="module" src="./index.ts"></script>
+  <body class="container">
+    <header class="header font-header">📚 Books app</header>
```

Comment:
You may want to wrap you heading in typography: 
```suggestion
    <header class="header font-header">
      <h1> 📚 Books app</h1>
    </header>
```
[Header ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header)MDN docs :)